### DISCUSSION TITLE
关于 ProTable API 的几个问题

### DISCUSSION THREAD

**Discussion by yingzhi0808** [ant-design/pro-components#1418] [Q&A]
1. toolbar.actions 和 toolBarRender 的功能是不是重复了
2. toolbar.title 和 headerTitle 的功能是不是重复了
3. toolbar 和 options 的功能是不是重复了
4. . 文档中的 toolbar.description 好像没效果

**Comment by CaitlynZ** [comment]
toolbar.description 没效果的问题 我用了subtitle替代，可以自己调一下样式让他符合你对description的样式期待
